channelfinder/cfUpdate/CFUpdateIOC.py

```diff
@@ -226,7 +226,7 @@ def ifNoneReturnDefault(object, default):
     """
     if the object is None or empty string then this function returns the default value
     """
-    if object == None and object != "":
+    if object is None and object != "":
```

Comment:
This combination doesn't really make sense. Based on the docstring, it should be `if object is None or object == ""`